docs/tutorial_bcc_python_developer.md

```diff
@@ -616,7 +616,7 @@ bpf_text = """
 #include <uapi/linux/ptrace.h>
 int do_trace(struct pt_regs *ctx) {
     uint64_t addr;
-    char path[128];
+    char path[128]={0};
```

Comment:
docs should follow the coding convention that tools/* do.
Please use 'char path[128] = {};'... note spaces...
Also this init is not really needed. Several releases ago bpf_probe_read() started to accept uninitialized memory, so it's best to avoid zero init and use it only on old kernels.